metc-wastewater-covid-monitor/server.R

```diff
@@ -436,6 +436,7 @@ server <- function(input, output) {
             family = font_family_list,
             color = councilR::colors$suppBlack
           ),
+          rangemode = "nonnegative",
```

Comment:
The same `rangemode = "nonnegative"` line appears 3 lines below.  I don't think you need both...  :-)